sql/src/benchmarks/java/io/crate/operation/projectors/GroupingProjectorBenchmark.java

```diff
@@ -87,20 +85,21 @@ public void testGroupByMinBytesRef() throws Exception {
             .createInjector().getInstance(Functions.class);
 
         InputCollectExpression keyInput = new InputCollectExpression(0);
-        List<Input<?>> keyInputs = Arrays.<Input<?>>asList(keyInput);
+        List<Input<?>> keyInputs = Arrays.asList(keyInput);
         CollectExpression[] collectExpressions = new CollectExpression[]{keyInput};
 
         FunctionIdent minStringFuncIdent = new FunctionIdent(MinimumAggregation.NAME,
-            Arrays.<DataType>asList(DataTypes.STRING));
+            Arrays.asList(DataTypes.STRING));
```

Comment:
please avoid unrelated cleanups - these can make rebasing hard for us if we have changes in the same files